sdk/identity/ci.yml

```diff
@@ -31,16 +31,14 @@ extends:
   template: ../../eng/pipelines/templates/stages/archetype-sdk-client.yml
   parameters:
     ServiceDirectory: identity
-    AdditionalTestMatrix:
-      Linux_Python35_msal:
-        Pool: Azure Pipelines
-        OSVmImage: 'ubuntu-18.04'
-        PythonVersion: '3.5'
-        CoverageArg: ''
-        RunForPR: false
-        InjectedPackages: 'git+https://github.com/AzureAD/microsoft-authentication-library-for-python@dev'
-
 
     Artifacts:
     - name: azure-identity
       safeName: azureidentity
+
+    ${{ if ne(variables['Build.Reason'], 'PullRequest') }}:
```

Comment:
Does this mean we also don't want to be able to trigger this job via a comment?

Reply:
Correct, this matrix entry is currently set up in master to not run for PR.